chore(rn): Share integrations documentation with javascript (#7009)
getsentry/sentry-docs: src/platforms/react-native/configuration/integrations/default.mdx

---
title: Default Integrations
excerpt: ""

> description: "Learn more about system integrations Dedupe, FunctionToString, Breadcrumbs, LinkedErrors, and HttpContext (UserAgent), that are enabled by default to integrate into the standard library or the interpreter itself."
> ---

Reviewer: ```suggestion
description: "Learn more about system integrations: Dedupe, FunctionToString, Breadcrumbs, LinkedErrors, and HttpContext (UserAgent), which are integrated into the standard library or the interpreter itself by default."
---
```
